packages/core/src/storages/utils.ts

```diff
@@ -22,3 +23,16 @@ export async function purgeDefaultStorages(config = Configuration.getGlobalConfi
         await client.purge?.();
     }
 }
+
+/**
+ * Easily create and manage state values. All state values are automatically persisted.
+ *
+ * Values can be modified by simply using the assignment operator.
+ *
+ * @param name The name of the store to use.
+ * @param defaultValue If the store does not yet have a value in it, the value will be initialized with the `defaultValue` you provide.
+ */
+export async function useState<T extends Dictionary = Dictionary>(name: string, defaultValue = {} as T) {
+    const kvStore = await KeyValueStore.open(null, { config: Configuration.getGlobalConfig() });
```

Comment:
you should be able to provide the config via parameter. maybe even the KVS name could be a parameter, so you could use a named storage if you want?